content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics.md

```diff
@@ -21,7 +21,7 @@ With topics, you can explore repositories in a particular subject area, find pro
 
 ![Screenshot of the github/docs repository. In the right sidebar, three topics are outlined in dark orange: "docs," "hacktoberfest," and "works-with-codespaces."](/assets/images/help/repository/os-repo-with-topics.png)
 
-To browse the most used topics, go to https://github.com/topics/.
+To browse the most used topics, go to {% ifversion fpt or ghec %}https://github.com/topics/{% elsif ghec or ghes %}`https://HOSTNAME/topics/`{% endif %}.
```

Comment:
```suggestion
To browse the most used topics, go to {% data variables.product.product_url %}/topics/.
```

Reply:
Hey @lecoursen , good to see you as well 👋 :octocat: 

It looks like this variable use may have broke the URLs on FPT/GHEC and then gives a bit of a weird format for GHES.

> GHEC - No hyperlink

![image](https://github.com/user-attachments/assets/f26c3510-59fd-4f74-b957-84751c912ead)

> GHES - No code format and/or no `https://` (though I guess you can deploy GHES with `http`, to be fair)

![image](https://github.com/user-attachments/assets/b12167e5-8571-4a7b-a2e6-8dab6327f000)

---

I'll let you suggest another change if you prefer another format. For reference, I used the GHES template found in [about-remote-repositories](https://docs.github.com/en/enterprise-server@3.13/get-started/getting-started-with-git/about-remote-repositories#about-remote-repositories)